training/cogvideox_image_to_video_lora.py

```diff
@@ -799,12 +799,15 @@ def load_model_hook(models, input_dir):
                 # (this is the forward diffusion process)
                 noisy_video_latents = scheduler.add_noise(video_latents, noise, timesteps)
                 noisy_model_input = torch.cat([noisy_video_latents, image_latents], dim=2)
-
+                model_config.patch_size_t if hasattr(model_config, "patch_size_t") else None,
```

Comment:
This seems wrong. No assignment to a variable. Is this expected?

Reply:
Forget to delete early edit. Fixed it